apps/frontend/src/index.tsx

```diff
@@ -55,7 +55,7 @@ createRoot(document.querySelector("#root") as HTMLElement).render(
 							},
 							{
 								element: (
-									<ProtectedRoute>
+									<ProtectedRoute isAvailableWithoutPermission>
```

Comment:
```suggestion
									<ProtectedRoute>
```
ProtectedRoute cannot be available without permissions

Reply:
However, this page must be available to authorised users who have not been assigned additional permissions, but not to others.

Reply:
we can simplify the logic by checking if routePermissions are empty. this would ensure that authenticated users can access the page without requiring additional permissions while still redirecting unauthenticated users to the sign-in page.